src/codeu/chat/DerbyStore.java

```diff
@@ -202,11 +210,10 @@ public void addConversation(Conversation c) throws SQLException {
 		addConversation.setString(5, removeCharsInUuid(c.firstMessage.toString()));
 		addConversation.setString(6, removeCharsInUuid(c.lastMessage.toString()));
 		addConversation.executeUpdate();
-		
 	}
 	
 	@Override
-	public void updateConversation(Conversation c) throws SQLException {
+	public void updateConversationMessages(Conversation c) throws SQLException {
```

Comment:
Some javadoc (maybe on the interface) explaining when these functions should be called would be helpful.